metron-deployment/inventory/full-dev-platform/group_vars/all

```diff
@@ -78,7 +79,7 @@ sensor_test_mode: True
 install_pycapa: False
 install_bro: True
 install_snort: True
-install_yaf: True
+install_yaf: False
```

Comment:
Are some missing? I did intend to have the probe off as the topology isn't started so it's a waste of memory and I couldn't get the cluster service started with it running.